src/components/Book.js

```diff
@@ -1,12 +1,25 @@
 import PropTypes from 'prop-types';
+import { actionRemove } from '../actions';
+import store from '../store';
 
-const Book = ({ id, title, category }) => (
-  <tr>
-    <td>{id}</td>
-    <td>{title}</td>
-    <td>{category}</td>
-  </tr>
-);
+const Book = ({ id, title, category }) => {
+  const removeBook = () => {
+    store.dispatch(actionRemove({ id }));
+  };
```

Comment:
- Implement this function in the BookList component and pass it here as a prop to the Button component